### DISCUSSION TITLE
Trouble downloading Loom videos

### DISCUSSION THREAD

**Discussion by Lucy481** [aclap-dev/video-downloadhelper#658] [Help and Q&A]
### Your question

I was trying to download a few different videos that were on Loom. I got the error "Download failed" (I tried doing a search for others with this issue but don't understand what I found in the search and didn't see anything pointing to Loom. 

Here is the copied bug detail reports:

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tls @ 0000020eb57e2dc0] IO error: Error number -10054 occurred\r\n[hls @ 0000020eb575a9c0] Error when loading first segment 'https://luna.loom.com/id/764eef485e9044129dc0c59aed479504/rev/64d9db6063feb7d00002025f3ecca226687d857ac45fbf5026a8a17e072bfb872/resource/hls/764eef485e9044129dc0c59aed479504-audio-0.ts'\r\n[in#0 @ 0000020eb575a800] Error opening input: Invalid data found when processing input\r\nError opening input file https://luna.loom.com/id/764eef485e9044129dc0c59aed479504/rev/64d9db6063feb7d00002025f3ecca226687d857ac45fbf5026a8a17e072bfb872/resource/hls/mediaplaylist-audio.m3u8?Signature=olOweXF3YrFWKf-JKOWwi5FSVg61QLCawuvh8tKeach9Dg8UYoeDVKov713wqZz2CLrrMPlZ52qBDTzFVSVIjigzUbEWruUvhxBOg7WrpbE0lCDnLa4AJffWY2Uf9bHexIWVxx09I1Jgj5mpmKFO~XCLnfIm2GR0iaa-Y4Lt5jb2~VKrQbxc~lybaTksv48wTo~JxvIeQUOT~TK0ugfYPmhSB9DV2VmpVlgc5hcyIsn-aYS~vaWL3NbNQ3WnzpP05~jRIBiuijR-TrfytA~1Xr2vDUkg9vVwDE56u9u-bSbZnm1sulRvgnoJj7aE4L3lYLu5jlpZxfhKkGWw8owr2w__&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9sdW5hLmxvb20uY29tL2lkLzc2NGVlZjQ4NWU5MDQ0MTI5ZGMwYzU5YWVkNDc5NTA0L3Jldi82NGQ5ZGI2MDYzZmViN2QwMDAwMjAyNWYzZWNjYTIyNjY4N2Q4NTdhYzQ1ZmJmNTAyNmE4YTE3ZTA3MmJmYjg3Mi9yZXNvdXJjZS8qIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoxNzE2NzUyODg4fX19XX0_&Key-Pair-Id=APKAJQIC5BGSW7XXK7FQ.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_b65468aa-8ec9-4fc1-9be5-871ec746f4c0",
    "downloadable_id": "downloadable_6378536304692147"
  }
}

### Addon & CoApp version and details

Version 9.0.1.13 (stable), target mozilla, locale en-US.
and 
Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I tried to visit the second URL in your error message.  That URL includes m3u8 within it, which would indicate to me that it is an HLS manifest, a master manifest if we're lucky, but at least a stream manifest.  But it turns out it's not a manifest at all but rather an XML file.  That's useless.

What would be more useful is a URL of a specific web page on which a video is hosted that VDH is having trouble downloading.  Normally, such a URL would appear in your problem report in the tag page_url.  But yours just says undef.  That's no help.  Saying you're having trouble with Loom is too vague.  Give me something I can work with.  That would be a specific URL of a specific web page on which a video is hosted.  Make it easy for me to help you.  Post that URL & let's see what I can do for you.
